realtime/__init__.py

```diff
@@ -1,6 +1,7 @@
 __version__ = "2.0.0"
 
 from realtime.channel import CallbackListener, Channel
+from realtime.client import RealTimeClient
```

Comment:
consider `realtimeClient` to mirror the swift lib https://github.com/supabase-community/realtime-swift

Reply:
note that that repo you linked is archived, swift lib is a monorepo and realtime is on https://github.com/supabase/supabase-swift/tree/main/Sources/Realtime